src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/impl/strategies/adapters/FizzBuzzOutputStrategyToFizzBuzzExceptionSafeOutputStrategyAdapter.java

```diff
@@ -8,7 +8,7 @@
  */
 public final class FizzBuzzOutputStrategyToFizzBuzzExceptionSafeOutputStrategyAdapter implements FizzBuzzExceptionSafeOutputStrategy {
 
-	private  final FizzBuzzOutputStrategy _fizzBuzzOutputStrategy;
+	private   final FizzBuzzOutputStrategy _fizzBuzzOutputStrategy;
```

Comment:
There should only be one space following the keyword `private` rather than three spaces. Please revise your PR to reflect the code formatting seen in existing instances of `private final` such as on [line 17 in the LoopComponentFactory](https://github.com/EnterpriseQualityCoding/FizzBuzzEnterpriseEdition/blob/4922c077c07a3744ae67ee8a932786f15bf57411/src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/impl/factories/LoopComponentFactory.java#L17).

My proposed change to your PR looks like this:
```java
private final FizzBuzzOutputStrategy _fizzBuzzOutputStrategy;
```